### PR TITLE
Wire up codecov

### PR DESCRIPTION
**Pull request by lfender6445**
# [codecov dashboard](https://codecov.io/github/rentpath/react-ui/commit/dad3f600adb894ac34a18d07050ea3b73d1fa8a9)

## todo later (@colinrymer maybe you can help me w this)
- [ ] try out slack integration

## check out the browser extension 

https://youtu.be/d6wJKODB8_g

## badge example (this branch)
[codecov](https://codecov.io/gh/rentpath/react-ui)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,18 +15,19 @@
     "postinstall": "lerna bootstrap",
     "prepublish": "lerna run clean --yes && lerna run build",
     "publish-packages": "lerna publish --exact --conventional-commits --registry=https://registry.npmjs.org/",
-    "report-coverage": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls",
     "storybook": "start-storybook -p 6006 -c .storybook -s ./.storybook/dist",
     "test": "./node_modules/.bin/jest --coverage",
     "test:ci": "yarn run test --maxWorkers=2 && yarn run report-coverage",
-    "test:coverage": "yarn run test && yarn run report-coverage"
+    "test:coverage": "yarn run test && ./node_modules/codecov/bin/codecov -t 9729bc93-20e5-42cb-aa55-a5308bd88f52",
```
Comment (colinrymer) [inline package.json]: We'll want to pull the codecov token out of here and read from an environment variable.

Comment (lfender6445) [reply package.json]: why? the coverage reports are only uploaded if someone runs coverage command. 

if react-ui is oss, shouldn't anyone be able to generate a codecov diff?

Comment (lfender6445) [reply package.json]: hmm this is interesting https://github.com/codecov/codecov-node/issues/64

File: codecov.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  range: "70...100"
+
+  status:
+    project:
+      default: on
+    patch:
+      default: on
+    changes:
+      default: off
+  # notify:
```
Comment (justinrex) [inline codecov.yml]: any reason to leave these as comments instead of removing them?

**Pull request closed without merging**